apps/element-storybook/.storybook/main.ts

```diff
@@ -5,7 +5,8 @@ import { StorybookConfig } from '@storybook/react-vite';
 const config: StorybookConfig = {
   stories: [
     '../src/*.mdx',
-    '../src/**/*.@(stories.@(js|jsx|ts|tsx|mdx))',
+    '../src/**/*.mdx',
+    '../src/**/*.@(stories.@(js|jsx|ts|tsx))',
```

Comment:
do we need to update all the package readme wrappers to remove `stories` from the name?

Reply:
nvm, seems to work fine in the preview

Reply:
Yeah, I renamed all the .stories.mdx files when I upgraded storybook.